src/client.js

```diff
@@ -69,7 +69,7 @@ class CommandoClient extends discord.Client {
 		// Fetch the owner(s)
 		if(options.owner) {
 			this.once('ready', () => {
-				if(options.owner instanceof Array || options.owner instanceof Set) {
+				if(Array.isArray(options.owner) || options.owner.toString() == "[object Set]") {
```

Comment:
i could understand using `Array.isArray` but why `.toString() === "[object Set]"`, feels kinda wacky

could even just check if the item is iterable like
`typeof options.owner[Symbol.Iterator] === 'function'` (maybe a better way not sure)

Reply:
A Map would also be iterable, but not valid in this case.